profiling/src/profile/mod.rs

```diff
@@ -433,6 +438,12 @@ impl Profile {
             .insert(interned_span_id, interned_endpoint);
     }
 
+    pub fn add_endpoint_count(&mut self, endpoint: Cow<str>, value: i64) {
```

Comment:
If you're going to convert the endpoint to a `String` anyway, you should probably take a String as argument